include/spm/evt_door.h

```diff
@@ -109,8 +110,13 @@ EVT_DECLARE_USER_FUNC(evt_door_set_map_door_descs, 1)
 EVT_DECLARE_USER_FUNC(evt_door_enable_disable_map_door_desc, 2)
 
 UNKNOWN_FUNCTION(evt_door_add_map_door_desc_to_unk_mask);
-UNKNOWN_FUNCTION(func_800e45c8);
-UNKNOWN_FUNCTION(func_800e468c);
+
+// evt_door_set_event(char *door, int unknown, EvtScriptCode * script)
+EVT_DECLARE_USER_FUNC(evt_door_set_event, 3)
+
+// evt_door_toggle(int flags) 
+EVT_DECLARE_USER_FUNC(evt_door_toggle, 1);
```

Comment:
The naming convention for these is usually `onoff` instead of toggle to match the ttyd map's style. What's it actually toggling, the door existing or being openable?

Reply:
being openable iirc

Reply:
Probably worth alluding to that in the name too then

Just noticed you called the parameter flags too, I'd use that name when it's taking an actual bitmask (like in most flag_onoff functions), here it's just a bool

Reply:
what name would you suggest then? The function is just used to unlock doors that say "looks like it won't open" but if I say something like evt_door_lock_onoff it sounds like im talking about doors with an actual key

Reply:
I guess something like evt_door_openable_onoff?

Reply:
alright!